pkg/handler/collector/deps_dev/deps_dev_test.go

```diff
@@ -63,127 +64,135 @@ func Test_depsCollector_RetrieveArtifacts(t *testing.T) {
 		interval           time.Duration
 		wantErr            bool
 		errMessage         error
-	}{{
```

Comment:
why are there so many format changes randomly?

Reply:
This is the reason because of `gofumpt`. I have `zsh func` that will `fumpt` everything before the push.

```
function process_go_files() {
    # Change to the current directory (where the function is called from)
    cd "$(dirname "$0")"

    # Get the list of all changed files (including staged, unstaged, and untracked files)
    # The output will include the relative path to the files
    local changed_files=$(git status --porcelain | awk '{print $2}')

    # Filter the list to include only .go files
    local go_files=$(echo "$changed_files" | grep '\.go$')

    # Loop through each .go file
    for file in ${(f)go_files}; do
        # Convert the relative path to a full path
        local full_path=$(realpath "$file")

        # Run the gci command with specified options
        gci write -s "standard,default" "$full_path"

        # Run the gofmt command to format the Go file
        gofmt -w "$full_path"

        gofumpt -w "$full_path"

        # Run the goimports command to update imports in the Go file
        goimports -w "$full_path"

        git add "$full_path"

        # Print the processed file's full path
        echo "Processed: $full_path"
    done
    make lint
    make fmt
}
```